### ISSUE TITLE
CLOB Error on Admissions table join 

### ISSUE THREAD

**Issue by tnaumann** [mimic2/v3.0#91]
@mghassem discovered that the following query causes an error.

Query:
```sql
select distinct a.subject_id, a.hadm_id, a.first_service, a.adm_diagnosis
from mimic2v30.d_patients d, mimic2v30.admissions a
where d.subject_id = a.subject_id;
```

Error:
```
ORA-00932: inconsistent datatypes: expected - got CLOB
00932. 00000 -  "inconsistent datatypes: expected %s got %s"
*Cause:    
*Action:
Error at Line: 1 Column: 59
```

However, the query completes when:

* `a.adm_diagnosis` is removed, or
*  the join condition is removed (i.e. we're only querying from the admissions table).

**Comment by tompollard** [comment]
Thanks Tristan. This should be fixed in the release version of MIMIC-II V3.0 because data types have been made consistent, so I'm closing the issue for now. Please could you report other pre-release issues at: https://github.com/MIT-LCP/building_mimic2_v3/issues ?

Cheers,
Tom

**Comment by alistairewj** [comment]
Yes, distinct fails on clob datatypes. We replaced diagnosis with a varchar2(200) (no diagnoses were longer than 200 characters).
